client/extensions/woocommerce/state/wc-api/shipping-zones/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_API_FETCH_SHIPPING_ZONES,
+	WOOCOMMERCE_API_FETCH_SHIPPING_ZONES_SUCCESS,
+} from '../../action-types';
+
+export const LOADING = 'LOADING';
+
+// TODO: Handle error
+
+export default {
+	[ WOOCOMMERCE_API_FETCH_SHIPPING_ZONES ]: ( state ) => {
+		return { ...state,
+			shippingZones: LOADING,
```

Comment:
This is an interesting approach. I've usually just held an empty set until data existed.